lib/alerts.py

```diff
@@ -8,6 +8,15 @@
 class Alerts(SerialisedModelCollection):
     model = Alert
 
+    def __init__(self, items):
+        sorted_items = sorted(
+            items,
+            reverse=True,
+            key=lambda item: item['starts_at']
```

Comment:
If you implement `__lt__` on the `Alert` class then you make the sorting reusable and don’t need the lambda or `reverse` here.

See this example from the admin app: https://github.com/alphagov/notifications-admin/blob/0363181daee439fe742fab6e1584d6a248b8df85/app/models/broadcast_message.py#L49-L62

Reply:
We'd still want `reverse`, as we specifically want to show the list in date descending order. Doing it at the point of presentation is more explicit, though. I've rewritten the final commit to do that.

How does it look now?

Reply:
Moving it into the Jinja is nice 👍🏻